jenkins/ci.suse.de/cloud-socok8s.yaml

```diff
@@ -6,4 +6,5 @@
     repo-credentials: c2350527-476a-45df-b406-84f028614682
     jobs:
       - '{name}-integration'
+      - '{name}-nightly-trigger'
```

Comment:
Why not one job for each deployment instead of a multijob?

Reply:
My question is already outdated by https://github.com/SUSE-Cloud/automation/pull/3429 . Should this PR be closed?

Reply:
I don't think it makes sense to have two different jobs where the only difference is the default value of the deployment parameter.